### DISCUSSION TITLE
Spark 2 Supported

### DISCUSSION THREAD

**Discussion by jbinedina** [soundshed/soundshed-app#74] [General]
Hi - is the new Spark 2 supported with either the web app or windows app? Thanks!

**Comment by webprofusion-chrisc** [comment]
Hi, I don't own a Spark 2 so haven't been able to try it out but I've heard that it works for selecting the first 4 presets but that Spark 2 offers 8 presets and so you don't get the full access to those currently.

**Comment by Dweinbach** [comment]
my initial observations for Spark 2:
a) for me, the windows app will see the amp but not retrieve presets. so the next two comments are for your hosted web app only:
b) I can confirm you can only pick the 1st 4 presets (but it will show the others if you select them with the knob on the amp)
c) Once a preset is selected, I do not think the knobs and switches actually affect the sound the amp is producing (unless it is very, very subtle) (I didn't "store settings" - I am assuming the changes should occur without doing that?)

**Comment by RafalSkolasinski** [comment]
For me hosted app seems to work but desktop app does not... Maybe a new build is needed?
